src/auto.ts

```diff
@@ -6,34 +6,23 @@ import { getSnapshot } from "./getSnapshot";
 
 export const auto = async (
   task: string,
-  config: { page: Page; test: Test },
+  config: { page: Page; test?: Test },
   options?: StepOptions
 ): Promise<any> => {
-  if (!config || !config.page || !config.test) {
+  if (!config || !config.page) {
     throw Error(
-      "The auto() function is missing the required `{ page, test }` argument."
+      "The auto() function is missing the required `{ page }` argument."
     );
   }
 
-  const { test, page } = config as { page: Page; test: Test };
+  const { test, page } = config as { page: Page; test?: Test };
 
-  return test.step(`auto-playwright.ai '${task}'`, async () => {
-    if (task.length > MAX_TASK_CHARS) {
-      throw new Error(
-        `Provided task string is too long, max length is ${MAX_TASK_CHARS} chars.`
-      );
-    }
+  if (!test) {
+    return await runTask(task, page, options);
```

Comment:
I don't understand the intent here. You are marking `test` as optional, but then failing if it is not provided?

Reply:
It's not failing, it runs the `runTask`

Reply:
The condition here is to run the flow without the `test` param, (it returns the result immediately).

I've tested it locally and it works.

Reply:
Oh, I see. Sorry, misread. Hm. What's the reason for this change?

Like.. why do you want it? I am thinking this will backfire if I need to access `test` in order to log steps.

Reply:
No worries. I need this mainly to use playwright as a browser automation tool. Eg, not in a test scenario. This can be really useful to interact with websites using a real browser.

I get that it could be a bit annoying with logs for test, might I suggest to create a log helper function that will fail silently or log somewhere else if the test param is undefined? That way both with work. And it will be easy to maintain and add logs. 

If you don't add this I'll have to fork/create a new repo for it and we would have duplicated code and maintenance. I really liked your work so I decided to fork and work from what you created.

Reply:
I've added an example to the readme.

Reply:
This makes sense. Thank you for the added context. Just waiting for checks to pass and will release.